packages/components/src/components/shared/FormText.tsx

```diff
@@ -46,16 +47,19 @@ const fontDescription = {
 }
 
 /**
- * Header
+ * Header text element commonly used in form components
  */
```

Comment:
Nitpicky, but this is aligned with the typing while others below are with the functional components.